diagnostics-app/src/app/app.module.ts

```diff
@@ -33,6 +34,12 @@ const initializeEventService = (
   return () => eventsService.Init();
 };
 
+const initializeRoutineV2Service = (
```

Comment:
I didn't notice before but just found that we have a style guide about function declaration

go/tsjs-style#function-declarations

Reply:
I see.

Let me change the occurence here, and I'll have another CL in the future that aligns all the conventions.

Reply:
Opened b/317948201